eis_toolkit/feature_importance/feature_importance.py

```diff
@@ -0,0 +1,48 @@
+import numpy as np
+import pandas
+import pandas as pd
+import sklearn.neural_network
+from sklearn.inspection import permutation_importance
+
+from eis_toolkit.exceptions import InvalidDatasetException
+
+
+def evaluate_feature_importance(
+    clf: sklearn.neural_network or sklearn.linear_model,
+    x_test: np.ndarray,
+    y_test: np.ndarray,
+    feature_names: list[str],
+    number_of_repetition: int = 50,
+    random_state: int = 0,
```

Comment:
Should the default be 0 or None? I thought None would be good default as it is so in Sklearn, but just asking

Reply:
this is an int should be any number I always set it to 0